mesheryctl/internal/cli/root/design/view.go

```diff
@@ -121,8 +121,8 @@ mesheryctl pattern view [pattern-name | ID]
 			// use the first match from the result when searching by pattern name
 			arr := dat["patterns"].([]interface{})
 			if len(arr) == 0 {
-				utils.Log.Error(ErrPatternNotFound())
-				utils.Log.Info(fmt.Sprintf("pattern with name: %s not found. Enter a valid pattern name or ID \n", pattern))
+				utils.Log.Error(ErrDesignNotFound())
+				utils.Log.Info(fmt.Sprintf("design with name: %s not found. Enter a valid design name or ID \n", pattern))
```

Comment:
Looks a duplicate to me, does this not already displayed in `ErrorDesignNotFound` error ?

Reply:
Noted. Removing this.

I just made a bulk rename and movement of commands from `pattern` to `design`. So I didn't notice these.